### ISSUE TITLE
Mismatch in translatable strings

### ISSUE THREAD

**Issue by milotype** [Hanziness/AnotherPomodoro#169]
**Is your feature request related to a problem? Please describe.**
The [Hungarian file](https://github.com/Hanziness/AnotherPomodoro/blob/develop/i18n/hu.js) has strings which are not within the English/Croatian translation files: lines 121 - 124.

Hungarian:
```
Line 121      methods: {
Line 122        github: 'Megtekintés GitHubon',
Line 123        buymeacoffee: 'Fejlesztő meghívása egy kávéra'
Line 124      },
```
**Describe the solution you'd like**
Either delete those lines from Hungarian if they are not needed, or add them to the other languages.

**Comment by Hanziness** [comment]
Yep, these are likely unused strings. Thank you for reporting them, I'll remove them in the next release!

**Comment by milotype** [comment]
À propos translations … if possible or applicable, use a translation platform for localizations – my favorite is [Weblate](https://weblate.org). There are others as well … That way you could automatically increase the number of languages/localizations … Translators are often not programmers, and do not understand GitHub, forks, commits, etc.

**Comment by Hanziness** [comment]
Yes, absolutely. I've already set up a Crowdin project - Weblate didn't seem very clear to me on whether I'm eligible to sign up for their open-source (libre) plan. Crowdin was easy to integrate with GitHub and the project's JSON translations, so for now I think I'll stick with that.

**Comment by milotype** [comment]
I've looked into the Crowding project. Please check how to handle plural forms of source strings, such as in the cases of:

- `hour | hours`
- `minute | minutes`

Several languages (especially slavic ones) need three or more forms. Croatian, for instance, needs three forms: `one`, `few`, `other`. Please add at least a note how to translate the source strings in cases of plural forms.

**Comment by Hanziness** [comment]
Sure! I'll see if I can add notes to the required strings. As per the [`vue-18n` documentation on pluralization](https://kazupon.github.io/vue-i18n/guide/pluralization.html), the order is `0 | 1 | more` or `1 | more` if there are only two strings. 

It also mentions in the [custom pluralization](https://kazupon.github.io/vue-i18n/guide/pluralization.html#custom-pluralization) section that one can implement custom pluralization rules for languages where this is more complex, but I'm afraid that can only be done using additional code - and I don't speak languages that would require this, so contributions are welcome if this is truly needed to properly display the plural forms of "minutes" and "hours" and no work-around or shorthand can do it in the current form.

**Comment by milotype** [comment]
That's why I proposed **Weblate** as the translation platform. In my experience, localizations with correct plural forms happen automatically. If you are unsure whether you are eligible to sign up for Weblate's open-source (libre) plan, simply contact support@weblate.org. In my experience, Michal Čihař responds quickly.

**Comment by Hanziness** [comment]
Sorry for getting back so late, I've just got back to working on this project more intensively :) This problem unfortunately has nothing to do with how the strings are translated (whether it's done in Crowdin/Weblate or directly in the translation files), it's the limitation of the localisation plugin in the Vue code. As I mentioned, the plugin *does* support custom pluralization rules, but they have to be manually implemented for languages that need more forms than `0 | 1 | more` or `1 | more` (in the case of English, eg. `car | cars`). 

I'm sure the lack of proper pluralization could be an annoyance for some, but I'd image it's a minor annoyance and not a lot of people are affected by it -- and the app can still be used in English, where this should not be a problem. Naturally if someone needs proper plural forms for their language, then they can submit their implementation to make it work (reference: https://kazupon.github.io/vue-i18n/guide/pluralization.html#custom-pluralization).

**Comment by Hanziness** [comment]
The issue of the mismatching strings was fixed in #206, as updated translations can now be imported from Crowdin.
